data/electronToChromium.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  "1.5": 54,
```

Comment:
All good - we can update and make a new version whenever, thanks for the head's up.